lib/rubocop/cop/style/special_global_vars.rb

```diff
@@ -91,15 +119,23 @@ class SpecialGlobalVars < Base
           :$* => %i[$ARGV ARGV]
         }
 
+        # Anything *not* in this set is provided by the English library.
+        NON_ENGLISH_VARS = Set.new(%i[$LOAD_PATH $LOADED_FEATURES $PROGRAM_NAME ARGV]).freeze
+
         PERL_VARS = ENGLISH_VARS.flat_map { |k, vs| vs.map { |v| [v, [k]] } }.to_h
 
         ENGLISH_VARS.merge!(ENGLISH_VARS.flat_map { |_, vs| vs.map { |v| [v, [v]] } }.to_h)
         PERL_VARS.merge!(PERL_VARS.flat_map { |_, vs| vs.map { |v| [v, [v]] } }.to_h)
+        BUILTIN_VARS = PERL_VARS.merge(
+          NON_ENGLISH_VARS
+            .select { |v| v.start_with?('$') }
```

Comment:
This is needed to workaround the :snowflake: `ARGV` otherwise we keep suggesting to use `ARGV` for `$*` with `use_builtin_names` which is not what we want here.